src/duct/middleware/web.clj

```diff
@@ -124,4 +124,8 @@
     b))
 
 (defmethod ig/init-key ::format [_ options]
-  #(mm/wrap-format % (deep-merge mc/default-options options)))
+  (let [malformed-handler (or (:malformed-handler options)
```

Comment:
A default handler can be overridden by config:
```clojure
{:duct.middleware.web/format {:malformed-handler #ig/ref :myapp.malformed-handler}
```

Reply:
Rather than use `or`, supply a default value instead.

Reply:
OK I'll try it.